recipes/libnghttp2/all/test_package_static_asio/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
You must remove  recipes/libnghttp2/all/test_package_static_asio and its content. CCI won't run it.

Reply:
Got it. I removed it.